iot/dotnet-iot-and-nanoframework/README.md

```diff
@@ -0,0 +1,67 @@
+--- 
+languages:
+- csharp
+products:
+- dotnet
+- dotnet-core
+- azure
+- azure-iot-hub
+page_type: sample
+name: "Compare .NET nanoFramework with .NET Core IoT"
+urlFragment: "compare-nanoframework-with-dotnet-core-iot"
+description: "A sample solution and documentation for comparing .NET nanoFramework and .NET Core IoT platforms. For this documentation same sample application with an identical scenario is created on both platforms, M5Stack with .NET nanoFramework and Raspberry Pi 2 (or above) with .Net Core IoT"
+---
+
+# Compare .NET nanoFramework with .NET Core IoT
+
+## What's .NET nanoFramework
+
+[.NET nanoFramework](https://www.nanoframework.net/) is a free and Open Source platform that enables the writing of managed code applications for constrained embedded devices. It is suitable for many types of projects including IoT sensors, wearables, academic proof of concept, robotics, hobbyist/makers creations or even complex industrial equipment.
+
+It makes the development for such platforms easier, faster and less costly by giving embedded developers access to modern technologies and tools used by desktop application developers. Developers can harness the powerful and familiar Microsoft Visual Studio IDE and their .NET C# knowledge to quickly write code without having to worry about the low-level hardware intricacies of a microcontroller. Desktop .NET developers will feel "at home" and are able to use their skills in embedded systems development, enlarging the pool of qualified embedded developers.
+
+This solution includes .NET Core IoT and .NET nanoFramework samples to compare two platforms and see max reusability. Any .NET developer can easily jump start to nanoFramework with their existing skills. There are some platform specific nuances, you'll learn those below.
```

Comment:
Try to avoid abbreviation like "max", better use "maximum".

Reply:
This is fixed